src/vl-mapactions-select-action.js

```diff
@@ -51,6 +56,7 @@ export class VlSelectAction extends VlMapAction {
       } else {
         element.style.cursor = '';
       }
+      this.map.render();
```

Comment:
is da performant dan, dattem alle lagen opnieuw gaat renderen?

Reply:
De rerender gebruiken we (zelfs in het verleden) nog op plaatsen, voorlopig zonder problemen maar het zal verder onderzocht worden in [#175](https://github.com/milieuinfo/webcomponent-vl-ui-map/issues/175) of uiteraard vroeger wanneer we performantie problemen zouden opmerken.